style.css

```diff
@@ -581,8 +415,8 @@ textarea {
 
 .error {
   position: absolute;
-  left: 10rem;
-  bottom: 1rem;
+  left: 5rem;
+  margin-top: 5rem;
```

Comment:
- [OPTIONAL] Kindly fix your design. The error should not go out of the container, it should still be displayed in the container. At this moment it doesn't look professional. :wink: 

    - **YOUR DESIGN**
     ![image](https://user-images.githubusercontent.com/87197412/210356683-289b8ebe-10f0-4b7c-8452-a5fd963efef1.png)